### ISSUE TITLE
[Bug]: auto_increment doesn't work properly

### ISSUE THREAD

**Issue by domingozhang** [matrixorigin/matrixone#1177]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 6b7d54b64e357f79f1b75d6fc1842a6d3ff5d990
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

create table t1 (id bigint auto_increment primary key, a int) auto_increment=9999999999;
insert into t1 values (null,1);
select * from t1;
+------+------+
| id   | a    |
+------+------+
| NULL |    1 |
+------+------+

### Expected Behavior

+-------------+------+
| id          | a    |
+-------------+------+
|  9999999999 |    1 |
+------+------+

**Comment by JinHai-CN** [comment]
Not support, until we have distributed transaction.

**Comment by m-schen** [comment]
it is same to issue #1142

**Comment by m-schen** [comment]
will solved at #1142. so I close this one.
